src/ui/public/vislib/styles/_tilemap.less

```diff
@@ -142,6 +142,10 @@
 .leaflet-control-attribution {
   background-color: @tilemap-leaflet-footer-bg !important;
   color: @tilemap-leaflet-footer-color !important;
+
+  p {
+    display: inline;
+  }
```

Comment:
running attribution through markdown wraps everything in a p block, causes linebreak